src/lib/CLValue/Key.ts

```diff
@@ -19,15 +15,47 @@ import {
   ToBytesResult,
   CLValueBytesParsers,
   CLValueParsers,
-  CLPublicKey,
   resultHelper,
-  ACCOUNT_HASH_TYPE,
-  BYTE_ARRAY_TYPE,
-  UREF_TYPE
+  KeyHashAddr,
+  HashParser,
+  KeyTransferAddr,
+  DeployHash,
+  EraInfo,
+  Balance,
+  KeyBid,
+  Withdraw,
+  KeyDictionary,
+  KeySystemEntityRegistry,
+  KeyEraSummary,
+  KeyUnbond,
+  KeyChainspecRegistry,
+  KeyChecksumRegistry,
+  KeyPackage,
+  ACCOUNT_HASH_PREFIX,
+  HASH_PREFIX,
+  UREF_PREFIX,
+  TRANSFER_PREFIX,
+  DEPLOY_HASH_PREFIX,
+  ERA_INFO_PREFIX,
+  BALANCE_PREFIX,
+  BID_PREFIX,
+  WITHDRAW_PREFIX,
+  DICTIONARY_PREFIX,
+  SYSTEM_ENTITY_REGISTRY_PREFIX,
+  ERA_SUMMARY_PREFIX,
+  UNBOND_PREFIX,
+  CHAINSPEC_REGISTRY_PREFIX,
+  CHECKSUM_REGISTRY_PREFIX,
+  BidAddr,
+  BidAddrParser
```

Comment:
Do we allow asterisk imports? Maybe it would be cleaner to just do `import * from './index'`?

Reply:
hm, do we use asterisk imports anywhere? its good idea but I would add this in 4.0.0 as standard